### ISSUE TITLE
okta_policy_password  issue with password_history_count

### ISSUE THREAD

**Issue by ahmei0** [okta/terraform-provider-okta#1457]
Enforce password history is shown ticked even though I have password_history_count = 0

I think once it is set to 4 and then setting it to zero or removing the attribute is not helping at all

![image](https://user-images.githubusercontent.com/62230464/220225880-927c715d-0473-480d-97fe-2a8a78b801dd.png)

```
resource "okta_policy_password" "myacc-password-pol" {
  name                   = "${local.name_prefix}"
  status                 = "ACTIVE"
  description            = "myacc wc password policy"
  groups_included = [
      okta_group.MyAccountCustomers.id
  ]
  password_min_symbol = 1
  password_exclude_username = false
  password_lockout_notification_channels = ["EMAIL"]
  recovery_email_token = 1440
  password_max_lockout_attempts = 5
  password_show_lockout_failures = true
  password_auto_unlock_minutes = 60
  question_recovery = "INACTIVE"
  password_history_count = 0
}
```

**Comment by monde** [comment]
I'm calling this a related duplicate of #1247 . We have issues with the okta-sdk-golang's implementation of integers as the primitive integer field and not integer pointers. The tags on that file are omitempty for json marshalling effectively making 0 values (0 history count, 0 priority, etc) not rendered in JSON.

**Comment by ahmei0** [comment]
Would setting this attribute help disable it?
________________________________

**Comment by monde** [comment]
This should be fixed now in [v3.44.0](https://github.com/okta/terraform-provider-okta/releases/tag/v3.44.0) release.
